tfhub_dev/assets/sayakpaul/deeplabv3-mobilenetv2/default/lite/1.md

```diff
@@ -1,19 +0,0 @@
-# Lite sayakpaul/deeplabv3-mobilenetv2/1/default/1
```

Comment:
The corresponding placeholder MD for this Lite MD files is not modified. Is this intended?